.github/workflows/code-style.yml

```
@@ -1,10 +1,27 @@
-name: Code style
+name: Format suggestions
 
 on:
   pull_request:
 
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
 jobs:
-  code-style:
+  format:
     runs-on: ubuntu-latest
     steps:
-      - uses: tkf/julia-code-style-suggesters@v1
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1
+      - run: |
+          julia  -e 'using Pkg; Pkg.add("JuliaFormatter")'
+          julia  -e 'using JuliaFormatter; format("."; verbose=true)'
+      - uses: reviewdog/action-suggester@v1
+        with:
+          tool_name: JuliaFormatter
```

I've followed the instructions under https://github.com/reviewdog/reviewdog#option-2-install-reviewdog-github-apps
So let's try this:
```suggestion
          github_token: ${{ secrets.REVIEWDOG_TOKEN }}
          tool_name: JuliaFormatter
```